### ISSUE TITLE
[lilliput] compiler warning (Trac #2078)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2070]
```text
/home/dschultz/Documents/combo/trunk/src/lilliput/private/minimizer/minuit/TMin
    uit.cxx: In member function ‘virtual void lilliput::TMinuit::mnprin(
        int32_t, double)’:
/home/dschultz/Documents/combo/trunk/src/lilliput/private/minimizer/minuit/TMinuit.cxx:5863:33:
    warning: comparison between signed and unsigned integer expressions [-
    Wsign-compare]
if (fCpnam[i-1].size()>= ic && fCpnam[i-1].substr(ic-1, 1) != " ") goto L16;
~~~~~~~~~~~~~~~~~~~^~~~~
```

This is technically external code, but at least it's easy to fix.

Migrated from https://code.icecube.wisc.edu/ticket/2078
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:51", 
    "description": "{{{\n/home/dschultz/Documents/combo/trunk/src/lilliput/private/minimizer/minuit/TMin\n    uit.cxx: In member function \u2018virtual void lilliput::TMinuit::mnprin(\n        int32_t, double)\u2019:\n/home/dschultz/Documents/combo/trunk/src/lilliput/private/minimizer/minuit/TMinuit.cxx:5863:33:\n    warning: comparison between signed and unsigned integer expressions [-\n    Wsign-compare]\nif (fCpnam[i-1].size()>= ic && fCpnam[i-1].substr(ic-1, 1) != \" \") goto L16;\n~~~~~~~~~~~~~~~~~~~^~~~~\n}}}\n\nThis is technically external code, but at least it's easy to fix.", 
    "reporter": "david.schultz", 
    "cc": "mase", 
    "resolution": "fixed", 
    "_ts": "1550067291068818", 
    "component": "combo reconstruction", 
    "summary": "[lilliput] compiler warning", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2017-08-10T16:24:25", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170811T08:22:56: kkrings changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170811T08:42:30: 
* kkrings commented:

> In changeset 157587:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157587"
> Declare local integer variable ic as unsigned; fixes #2070.
> ```


* kkrings changed resolution from "" to "fixed"
* kkrings changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:51: 
* olivas changed milestone from "December 2017" to ""
* olivas commented:

> Milestone December 2017 deleted
